packages/lib-algorand/index.ts

```diff
@@ -0,0 +1,89 @@
+import { DfnsApiClient, DfnsError } from '@dfns/sdk'
+import { GetWalletResponse, GenerateSignatureResponse } from '@dfns/sdk/types/wallets'
+
+import { encodeObj, Transaction, EncodedSignedTransaction, decodeObj } from 'algosdk'
+export type DfnsWalletOptions = {
+  walletId: string
+  dfnsClient: DfnsApiClient
+}
+
+const hexToBuffer = (hex: string): Buffer => {
+  return Buffer.from(hex.replace(/^0x/, ''), 'hex')
+}
+
+export const singleSigTransactionToEncoded = (tx: Transaction): EncodedSignedTransaction =>{
```

Comment:
with the overload, we don't have to export this function directly. that probably makes the dev experience a bit better.